posts/2023-06-27-23.0.0.6.adoc

```diff
@@ -0,0 +1,203 @@
+
+For more information, reference the link:{url-prefix}/blog/2023/06/29/rapid-startup-instanton.html[How to package your cloud-native Java application for rapid startup] blog post and the link:{url-prefix}/docs/latest/container-images.html[Faster startup with InstantOn] documentation.
+   
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC> 
+
+[#devTools]
+== Liberty Tools
+The Liberty Tools v23.0.6 are now available! Some highlights include diagnostic and hover support for MicroProfile and Jakarta EE Web Profile 9.x APIs as well as various Liberty config files.  The Liberty Tools are available for Visual Studio Code, IntelliJ IDEA, and Eclipse IDE.
+
+To get started with the Liberty Tools, you can obtain the link to install the tools for your favorite IDE from the link:{url-prefix}/start/[Get started with Open Liberty] page.
+
+For more information, reference the link:{url-prefix}/docs/latest/develop-liberty-tools.html[Develop with Liberty Tools] documentation.
+
+[#CVEs]
+== Security vulnerability (CVE) fixes in this release
+[cols="5*"]
+|===
+|CVE |CVSS Score |Vulnerability Assessment |Versions Affected |Notes
+
+|http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-28867[CVE-2023-28867]
+|7.5
+|Denial of service
+|17.0.0.3 - 23.0.0.5
+|Affects the link:{url-prefix}/docs/latest/reference/feature/mpGraphQL-1.0.html[mpGraphQL-1.0] and link:{url-prefix}/docs/latest/reference/feature/mpGraphQL-2.0.html[mpGraphQL-2.0] features
+|===
+
+For a list of past security vulnerability fixes, reference the link:{url-prefix}/docs/latest/security-vulnerabilities.html[Security vulnerability (CVE) list].
+
+
+[#bugs]
+== Notable bugs fixed in this release
+
+
+We’ve spent some time fixing bugs. The following sections describe just some of the issues resolved in this release. If you’re interested, here’s the  link:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A23006+label%3A%22release+bug%22[full list of bugs fixed in 23.0.0.6].
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/25283[JSF Container's Application.getWrapped returns null]
++
+The JSF Container features return `null` when `javax.faces.application.Application.getWrapped` method is called. 
++
+This issue has been resolved and the correct wrapped object is returned.
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/25210[DnsContextFactory not accessible in java 17]
++
+With Java 17 and Liberty trying to make a dns lookup fails. A ffdc is created with this message:
++
+```
+Exception = java.lang.IllegalAccessException
+Source = com.ibm.ws.jndi.internal.WASURLObjectFactoryFinder
+probeid = 81
+Stack Dump = java.lang.IllegalAccessException: class com.ibm.ws.jndi.internal.WASURLObjectFactoryFinder cannot access class com.sun.jndi.url.dns.dnsURLContextFactory (in module jdk.naming.dns) because module jdk.naming.dns does not export com.sun.jndi.url.dns to unnamed module @188cf1f7
+```
```

Comment:
is anything missing here? It seems like there should be some kind of resolution claim